### PR TITLE
 🔧 the security result class should not contain the result text

### PR DESCRIPTION
**Pull request by wei-lee**
### Motivation

At the moment, the security check result class contains a string to describe the error messages. This should not be part of the SDK.

### Description

Remove the text description of the security check results.

### Progress

- [x] Done Task

@aidenkeating @danielpassos @StephenCoady  mind taking a look?

### CODE REVIEW

**Comment by aidenkeating** [comment]
@wei-lee  👀

File: modules/security/deviceTrust/checks/DeviceLockCheck.swift [hunk 1]
```diff
@@ -21,9 +19,9 @@ import LocalAuthentication
     public func check() -> SecurityCheckResult {
         let deviceLockSet = LAContext().canEvaluatePolicy(.deviceOwnerAuthentication, error: nil)
         if deviceLockSet == false {
-            return SecurityCheckResult(self.name, false, self.failing)
+            return SecurityCheckResult(self.name, false)
         } else {
```
Comment (aidenkeating) [inline modules/security/deviceTrust/checks/DeviceLockCheck.swift]: We could probably put this on a single line now that it's simpler.

File: modules/security/deviceTrust/checks/NonJailbrokenCheck.swift [hunk 2]
```diff
@@ -20,9 +18,9 @@ import Foundation
      */
     public func check() -> SecurityCheckResult {
         if DTTJailbreakDetection.isJailbroken() {
-            return SecurityCheckResult(self.name, false, self.failing)
+            return SecurityCheckResult(self.name, false)
         } else {
```
Comment (aidenkeating) [inline modules/security/deviceTrust/checks/NonJailbrokenCheck.swift]: We could probably put this on a single line now.

**Comment by wei-lee** [comment]
@aidenkeating mind taking a look again?

**Pull request merged**